### ISSUE TITLE
Move all libraries to local

### ISSUE THREAD

**Issue by lmangani** [meething/meething#89]
Opening this low-fi task for consideration and volunteers to pick up.
We're currently loading files from various cdns - some are real slow and might benefit being local.

**Comment by jabis** [comment]
In favor of this, but I'd request the polyfill.io to be added for Edge - it only brings crap based on userAgent requesting, so mostly for Chrome it will bring in an empty comment, and with firefox a few shims, but Edge get's wealth of stuff from there, not excluding Sets,Weakmap, etc - that we can't have locally :) (I don't think it's even included in the index.html at the moment)

**Comment by lmangani** [comment]
Makes sense - let's `blacklist` or `whitelist` the current imports and see which we can adopt local?

**Comment by QVDev** [comment]
This means we manually need to update all libraries. Either loading the CDN is slow, or the client has a bad connection? I am OK, but think maintenance wise is gonna be a bit of a worry
